torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -256,6 +306,7 @@ def __init__(
         num_neighbors: NumNeighbors,
         edge_label_index: InputEdges = None,
         edge_label: OptTensor = None,
+        edge_time: OptTensor = None,
```

Comment:
Why for edge_time do we pass it as a tensor rather than passing it as an attribute name as we do for nodes?

Reply:
Good question. My thinking was that `edge_label_index` and `edge_label` are `Tensor` so it makes sense to have `edge_time` as a `Tensor` too.

Reply:
Hmm yeah I see that logic too. I feel it would be nice to be consistent, but no strong feeling on this.

Reply:
I think we need to pass it as a tensor here since the labels we are interested in are not necessarily part of the graph.

Reply:
That's true, but presumably the same could have been true for the node times?

Reply:
Nodes are a part of the graph (`HeteroData` object), so their time will always be a node attribute. 
But the edge we want to predict might not be in the graph.

Reply:
Oh good point. That makes sense thanks!